packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

```diff
@@ -281,7 +281,10 @@ export default VSelect.extend({
       const input = VTextField.options.methods.genInput.call(this)
 
       input.data = input.data || {}
-      input.data.attrs = input.data.attrs || {}
+      input.data.attrs = {
```

Comment:
```
input.data.attrs = input.data.attrs || {}
input.data.attrs.autocomplete = 'off'
```

Reply:
I just assumed that you should be able to turn browser autocomplete on, but that doesn't make sense at all :D Will update

Reply:
autocomplete is just passed through to the native input, the `input.data.attrs || {}` is to safeguard against a null or empty value. would likely throw an error if you spread a null value.